.travis.yml

```diff
@@ -40,4 +49,4 @@ addons:
     notification_email: none@example.com
     build_command_prepend: "./configure"
     build_command:   "make -j4"
-    branch_pattern: coverity_scan
+    branch_pattern: master|coverity_scan
```

Comment:
The docs say this value is a regular expression, so I _hope_ this works. I guess we’ll only find out when we push this to `master` or `coverity_scan`.